tests/test_blob_repairer.py

```diff
@@ -408,6 +416,74 @@ def test_check_index_non_valid(_mocked_exists,
     assert blob_repairer.stat.index_malformed_headers == 0
 
 
+@pytest.mark.parametrize('callee', [
+    BlobRepairer.recover_index,
+    BlobRepairer.recover_blob,
+    BlobRepairer.copy_valid_records,
+])
+@mock.patch('eblob_kit.is_destination_writable', return_value=True)
+@mock.patch(OPEN_TO_PATCH, new_callable=mock.mock_open)
+@mock.patch('eblob_kit.IndexFile', autospec=True)
+@mock.patch('eblob_kit.DataFile', autospec=True)
+@mock.patch('os.path.exists', return_value=True)
+def test_fix_destination_writable(_mock_exist,
+                                  mocked_data_file,
+                                  _mocked_index_file,
+                                  _mocked_open,
+                                  _mocked_is_writable,
+                                  mocker,
+                                  callee):
+    """Check for exception if destination not writable.
+
+    Checks for `copy_valid_records`, `recover_index` and `recover_blob`.
+
+    """
+    blob_repairer = BlobRepairer('.')
+
+    assert blob_repairer.valid
+    assert not blob_repairer.stat.index_order_error
+
+    if isinstance(callee, types.FunctionType):
```

Comment:
Is it a correct condition? What if there will be another static method with different arguments?

Reply:
Not sure that correct, but one of the first I've found on SO:
https://stackoverflow.com/questions/8727059/python-check-if-method-is-static
and not selected by TC answer, but with most votes.

In first version of tests, it was separate version for each method, but code was bloated with all tests are almost completely identical, so I ended up with current version.

Reply:
How about?
```suggestion
    if callee == BlobRepairer.recover_index:
```

Reply:
Yep, that do a trick.